docs/content/Workspace/Inspecting-Queries.mdx

```diff
@@ -5,55 +5,155 @@ category: Workspace
+  highlight="inset(13% 6% 79% 88% round 10px)"
+  src="https://ucarecdn.com/691e7cfa-e7fd-4df1-8cfa-f8f0d74634f6/"
+/>
 
-<div style="text-align: center">
-  <img
-    src="https://ucarecdn.com/a242026b-3a99-46fc-8803-147559985543/"
-    style="border: none"
-    width="100%"
-  />
-</div>
+## Exploring queries
 
-## Pre-aggregations
+You can switch between [queries made to the API](#api-queries) and
+[queries made to connected data sources](#database-queries) by switching between
+the <Btn>API</Btn> and <Btn>Database</Btn> tabs.
 
-When inspecting queries on the <Btn>Queries</Btn> page, you can now also see
-whether the query used a pre-aggregation:
+<Screenshot
+  highlight="inset(20% 64% 72% 20% round 10px)"
+  src="https://ucarecdn.com/44bb6d8d-5943-4978-8b15-73e1e43fd988/"
+/>
 
-![Query showing pre-aggregation lightning mark|674x500](https://ucarecdn.com/ebdfee8c-37f5-4bc3-8ed4-80f58864012c/)
+### <--{"id" : "API and Database"}--> All Queries and Top Queries
 
-If you click a query from the list, we now provide more relevant information,
-like if the query built a pre-aggregation or not:
+Clicking <Btn>All Queries</Btn> will show all queries in order of recency,
+while&nbsp;<Btn>Top Queries</Btn> will show the most-frequently executed
+queries:
 
-![Query request details|690x428](https://ucarecdn.com/93847927-1388-4e14-9782-ec0b6992b993/)
+<Screenshot
+  highlight="inset(27% 50% 65% 20% round 10px)"
+  src="https://ucarecdn.com/44bb6d8d-5943-4978-8b15-73e1e43fd988/"
+/>
 
-Cube Cloud also provides useful tips for pre-aggregation strategies and
-recommends the best pre-aggregation for a specific query. Click on any query on
-the <Btn>Queries</Btn> page and navigate to the <Btn>Pre-Aggregations</Btn> tab
-inside the query details page.
+### <--{"id" : "API and Database"}--> Filtering
 
-<div style="text-align: center">
-  <img
-    src="https://ucarecdn.com/aefcb733-12c2-4446-9520-c968062dd500/"
-    style="border: none"
-    width="100%"
-  />
-</div>
+You can also use filters to find problematic queries; you can filter by query
+duration, cache status, whether the query was accelerated, and by failing
+queries:
+
+<Screenshot
+  highlight="inset(34% 8% 60% 20% round 10px)"
+  src="https://ucarecdn.com/44bb6d8d-5943-4978-8b15-73e1e43fd988/"
+/>
+
+## API Queries
+
+To see an expanded view of a query, click on <Btn>❯</Btn> next to any query:
+
+<Screenshot
+  highlight="inset(1% 87% 6% 9% round 10px)"
+  src="https://ucarecdn.com/7ec40dc3-813d-4e3e-881b-dc01a6fdfb50/"
+/>
+
+You can also see how long the query took, whether the query was accelerated, and
+even view the raw query by clicking <Btn>❮/❯</Btn>:
+
+<Screenshot
+  highlight="inset(1% 2% 93% 68% round 10px)"
+  src="https://ucarecdn.com/e3c00115-49f9-48a6-8f2b-949c53523c9f/"
+/>
+
+To drill down on a specific query, click it to see more information.
+
+### <--{"id" : "API Queries"}--> Query
+
+The <Btn>Query</Btn> tab shows the raw JSON query sent to the Cube Cloud
+deployment.
```

Comment:
Does it also show a JSON query for a request through the SQL API? 🤨

Reply:
Interestingly, I'm not seeing any new items in the Queries page when making SQL API requests, could be a bug

Reply:
It seems like SQL API queries go directly to the "Database" queries tab instead of showing up in the "API" section 😕

Reply:
Confirmed with @pacofvf, SQL API queries show up as a JSON query too